hammurabi/mixins.py

```diff
@@ -140,6 +137,10 @@ def generate_pull_request_body(pillar) -> str:
         #  NOTE: The parameter type can not be hinted, because of circular import,
         #  we must fix this in the future releases.
 
+        def get_chained_rules(target, items):
```

Comment:
I would use `chain_rules`. Chained may also be correct. Tho I would only change it if we use the same terminology everywhere.